core/storage/classifier/gae_models_test.py

```diff
@@ -313,6 +339,47 @@ def test_get_deletion_policy(self) -> None:
             .get_deletion_policy(),
             base_models.DELETION_POLICY.NOT_APPLICABLE)
 
+    def test_get_export_policy(self) -> None:
+        expected_export_policy_dict = {
+            'exp_id': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'exp_version': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'state_name': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'algorithm_ids_to_job_ids': (
+                base_models.EXPORT_POLICY.NOT_APPLICABLE),
+            'created_on': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'deleted': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'last_updated': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+        }
+        self.assertEqual(
+            classifier_models.StateTrainingJobsMappingModel.get_export_policy(),
+            expected_export_policy_dict)
+
+    def test_get_model_from_exploration_attributes(self) -> None:
```

Comment:
Optional nit: I think I would suggest changing "from" to "based on", here and below. In my mind, "from" suggests converting the exploration attributes to a model (similar to how we use from_dict()).

Otherwise, this PR LGTM; thanks!